Match/MatchInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Immutable;
+using Common.PkmnModels;
+
+namespace Match
+{
+    public enum Side { Blue, Red }
+
+    public record MatchResult(Side? Winner);
+
+    public record MatchInfo(
+        IImmutableList<Pokemon> TeamBlue,
+        IImmutableList<Pokemon> TeamRed)
+    {
+        public string? Stage { get; init; }
+    // dotnet-format is drunk, see: https://github.com/dotnet/format/issues/805
+    public string? FieldEffect { get; init; }
+    public float Speed { get; init; } = 1f;
+}
```

Comment:
bracket alignment?

Reply:
see comment above, dotnet is doing weird stuff. 
If that's preferable I can fix it by marking this file as generated code in the editorconfig file instead, so dotnet-format won't touch it.

Reply:
no that should be fine